packages/flutter_hooks/test/use_platform_brightness_test.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:ui';
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+
+import 'mock.dart';
+
+void main() {
+  group('usePlatformBrightness', () {
+    testWidgets('returns initial value and rebuild widgets on change',
+        (tester) async {
+      tester.binding.window.platformBrightnessTestValue = Brightness.light;
```

Comment:
It seems like `.window.platformBrightnessTestValue` is deprecated. Could you update the tests to use the new solution?

Reply:
the new solution is `.platformDispatcher.platformBrightnessTestValue` but it's not on Stable channel yet.
Should I switch to beta channel and update it?

Reply:
@rrousselGit

Reply:
Sorry for the delay. Yes, please do so. There should be a new stable release soon

Make sure to update the Flutter SDK constraints in the pubspec.yaml too